ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_Samples/GAnalytics.cpp

```diff
@@ -50,7 +50,9 @@ GAnalytics::~GAnalytics()
 void GAnalytics::init()
 {
   generateClientId();
-  if (m_apiSecret.isEmpty() || m_measurementId.isEmpty())
+
+  // If this is an Esri daily build or a local user build, disable analytics
+  if (m_apiSecret.isEmpty() || m_measurementId.isEmpty() || m_apiSecret == "GANALYTICS_API_KEY" || m_measurementId == "GANALYTICS_STREAM_ID")
```

Comment:
Why would `m_apiSecret == "GANALYTICS_API_KEY"` ever happen? We should get to the bottom of that. This seems like a workaround.

Reply:
If `GANALYTICS_API_KEY` isn't defined then `QUOTE(GANALYTICS_API_KEY)` evaluates to `"GANALYTICS_API_KEY"`. If it's defined but `GANALYTICS_API_KEY` wasn't set on the command line, it evaluates to `""`. What I should do instead is move the defines outside of the daily build block and make sure it's defined regardless.